### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by vpiili**

### CODE REVIEW

File: src/components/IndexRoute.tsx [hunk 1]
```diff
@@ -3,11 +3,13 @@ Copyright Juhani Vähä-Mäkilä (juhani@fmail.co.uk) 2022.
 Licenced under EUROPEAN UNION PUBLIC LICENCE v. 1.2.
  */
 import React from 'react'
+import { useTranslation } from 'react-i18next'
 
 export default function IndexRoute() {
+  const { t } = useTranslation()
```
Comment (aegan123) [inline src/components/IndexRoute.tsx]:
```suggestion
  const { translation } = useTranslation()
```

File: src/components/LanguageSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  return (
+    <Wrapper>
+      <span className={`fi fi-${langToIcon[currentLanguage]} fis`}></span>
+      <Dropdown>
+        <p onClick={() => void i18n.changeLanguage('fi')}>
+          <span className={`fi fi-fi fis`}></span>
+        </p>
+        <p onClick={() => void i18n.changeLanguage('en')}>
+          <span className={`fi fi-us fis`}></span>
```
Comment (aegan123) [inline src/components/LanguageSelector.tsx:38]: fi-us?

File: src/components/IndexRoute.tsx (hunk 1 above)
Comment (vpiili) [reply src/components/IndexRoute.tsx]: `t` comes here from object destructuration. One can change this by either wrapping `useTranslation` hook or renaming destructured property (`{ t: translate } = useTranslation()`)

Comment (vpiili) [reply src/components/IndexRoute.tsx]: Imo `{ t }` is still best :)

File: src/components/LanguageSelector.tsx (hunk 2 above)
Comment (vpiili) [reply src/components/LanguageSelector.tsx:38]: That stands for flag-icons

**Pull request merged**